src/app/lecture/knowledge-node/learning-objects/arrange-task/arrange-task.component.html

```diff
@@ -0,0 +1,36 @@
+<div id="arrange-task" class="mat-card">
+  <h3 class="mat-h3">{{learningObject.text}}</h3>
```

Comment:
As you will see in the data, it would be helpful if this is markdown to support a better task definition.